FitnessClub/src/app/shared/create-workout/create-workout.component.scss

```diff
@@ -4,24 +4,24 @@
     color: white;
     padding-bottom: 200px;
     overflow: hidden;
-}
 
-.create-workout a {
-    display: inline-block;
-    margin-top: 20px;
-    width: 200px;
-    height: 50px;
-    background-color: #c9138c;
-    color: #000;
-    font-weight: bold;
-    text-decoration: none;
-    font-family: Arial, sans-serif;
-    font-style: initial;
-    line-height: 50px;
-    border-radius: 20px;
-}
+    a {
+      display: inline-block;
+      margin-top: 20px;
+      width: 200px;
+      height: 50px;
+      background-color: #c9138c;
+      color: #000;
+      font-weight: bold;
+      text-decoration: none;
+      font-family: Arial, sans-serif;
+      font-style: initial;
+      line-height: 50px;
+      border-radius: 20px;
 
-.create-workout a:hover {
-    background-color: #110e10;
-    color: #c9138c;
+      &:hover {
```

Comment:
please ensure you benefit from scss nested selectors rather that using plain CSS